lib/hammer_cli_foreman/compute_resource.rb

```diff
@@ -108,6 +108,30 @@ class DeleteCommand < HammerCLIForeman::DeleteCommand
       build_options
     end
 
+    class AvailableClustersCommand < HammerCLIForeman::ListCommand
+
+     resource :compute_resources, :available_clusters
+     command_name 'clusters'
+     desc _("Show available clusters")
+
+     build_options
+
+     validate_options do
+       any(:option_id, :option_name).required
+     end
+
+     def request_params
+       params = super
+       params['id'] ||= get_identifier
+       params
+     end
+
+     output do
+       field :name, _('Name')
+       field :id,   _('Id')
+     end
```

Comment:
This is indented only 1 space instead of 2

Reply:
Looks fine in Vim/Atom

Reply:
NM see what you mean and fixed,.